cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,52 @@
-describe('', () => {
-  before(() => {
+describe('Article flow', () => {
+  let user;
+  let article;
 
-  });
+  before(() => {
+    cy.visit(`/user/login`);
+    
+    cy.task('generateUser').then((generateUser =>{
+      user = generateUser;
+    }));
 
-  it('', () => {
+    cy.task('generateArticle').then((generateArticle =>{
+      article = generateArticle;
+    }));
 
   });
 
-  it('', () => {
-
+  it.skip('should create an article', () => {
```

Comment:
```suggestion
  it('should create an article', () => {
```

Reply:
you could miss critical error on the real project with that)

Reply:
fixed